examples/miscellaneous/plot_set_output.py

```diff
@@ -114,3 +115,20 @@
 # This resets `transform_output` to its default value to avoid impacting other
 # examples when generating the scikit-learn documentation
 set_config(transform_output="default")
+
+# %%
+# Use `config_context` context manager to adjust the transform output within the context
+# manager. Note that you should use the context manager around the `transform` or
+# `fit_transform` usage, Transformer constructed within the context manager will not
+# be affected.
```

Comment:
What do you think of something like the following:

```suggestion
# When configuring the output type with :func:`config_context` the
# configuration at the time when `transform` or `fit_transform` are
# called is what counts. Setting these only when you construct or fit
# the transformer has no effect.
```

Reply:
Sounds good to me - updated.